src/main/webapp/app/pages/curation/list/FirebaseList.tsx

```diff
@@ -66,7 +66,7 @@ function FirebaseList<T>({ path, itemBuilder, pushDirection, scrollOptions, filt
         return;
       }
       const items = (await get(ref(firebaseDb, path))).val();
-      if (!items && indices?.length !== 0) {
+      if (!items || indices?.length !== 0) {
```

Comment:
What's the reason for this change?

Reply:
`item` could be null coming out of firebase and it was crashing the page.

Reply:
Looks like screenshot tests aren't happy with this code...